contracts/MetadataViews.cdc

```diff
@@ -129,4 +129,38 @@ pub contract MetadataViews {
             return "ipfs://".concat(self.cid)
         }
     }
+
+    // A view to represent Media, a file with an correspoiding mediaType.
+    // media-type comes on the form of type/subtype as described here https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types
```

Comment:
Could we add this comment above the `mediaType` line instead of above the `Media` struct?

Reply:
Yes